x/bep3/types/expected_keepers.go

```diff
@@ -2,10 +2,11 @@ package types
 
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	authexported "github.com/cosmos/cosmos-sdk/x/auth/exported"
 	supplyexported "github.com/cosmos/cosmos-sdk/x/supply/exported"
 )
 
-// SupplyKeeper defines the expected supply Keeper
+// SupplyKeeper defines the expected supply keeper
```

Comment:
can be a separate PR, but we should follow the expected keepers pattern (see CDP, which was just updated, for exmaple).

Basically, add (noalias) to each type in `types/expected_keepers.go` and then in `module.go` change `NewAppModule` to take `types.AccountKeeper` and `types.SupplyKeeper` as args.

Reply:
Updated to expected keepers pattern in https://github.com/Kava-Labs/kava/pull/482/commits/dcceda48dc241fb7fae7cedea70bb3c67a0c0b69, https://github.com/Kava-Labs/kava/pull/482/commits/f1ef515f20f11020b84830a62cd0ff09e681b5a9